docs/scos/dev/front-end-development/202311.0/marketplace/table-design/table-column-type-extension/table-column-type-button-action.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Table Column Type Button Action
+description: This document provides details about the Table Column Type Button Action in the Components Library.
+template: concept-topic-template
+related:
+  - title: Table Column Type extension
+    link: docs/scos/dev/front-end-development/page.version/marketplace/table-design/table-column-type-extension/table-column-type-extension.html
+  - title: Table Column Type Autocomplete
+    link: docs/scos/dev/front-end-development/page.version/marketplace/table-design/table-column-type-extension/table-column-type-autocomplete.html
+  - title: Table Column Type Chip
+    link: docs/scos/dev/front-end-development/page.version/marketplace/table-design/table-column-type-extension/table-column-type-chip.html
+  - title: Table Column Type Date
+    link: docs/scos/dev/front-end-development/page.version/marketplace/table-design/table-column-type-extension/table-column-type-date.html
+  - title: Table Column Type Dynamic
+    link: docs/scos/dev/front-end-development/page.version/marketplace/table-design/table-column-type-extension/table-column-type-dynamic.html
+  - title: Table Column Type Image
+    link: docs/scos/dev/front-end-development/page.version/marketplace/table-design/table-column-type-extension/table-column-type-image.html
+  - title: Table Column Type Input
+    link: docs/scos/dev/front-end-development/page.version/marketplace/table-design/table-column-type-extension/table-column-type-input.html
+  - title: Table Column Type List
+    link: docs/scos/dev/front-end-development/page.version/marketplace/table-design/table-column-type-extension/table-column-type-list.html
+  - title: Table Column Type Select
+    link: docs/scos/dev/front-end-development/page.version/marketplace/table-design/table-column-type-extension/table-column-type-select.html
+  - title: Table Column Type Text
+    link: docs/scos/dev/front-end-development/page.version/marketplace/table-design/table-column-type-extension/table-column-type-text.html
+---
+
+This document explains the Table Column Type Button Action in the Components library.
+
+## Overview
+
+Table Column Button Action is an Angular Component that renders button using the `@spryker/button.action` component.
```

Comment:
Maybe it's better to use reference to the doc page, instead of `@spryker/button.action`

Reply:
Do we have the doc for `button.action`?